src/Features/Core/Diagnostics/Log/DiagnosticLogAggregator.cs

```diff
@@ -61,26 +63,30 @@ public AnalyzerInfo(DiagnosticAnalyzer analyzer, AnalyzerActions analyzerActions
                 CLRType = analyzer.GetType();
                 Telemetry = telemetry;
 
-                Counts[0] = analyzerActions.CodeBlockEndActionsCount;
-                Counts[1] = analyzerActions.CodeBlockStartActionsCount;
-                Counts[2] = analyzerActions.CompilationEndActionsCount;
-                Counts[3] = analyzerActions.CompilationStartActionsCount;
-                Counts[4] = analyzerActions.SemanticModelActionsCount;
-                Counts[5] = analyzerActions.SymbolActionsCount;
-                Counts[6] = analyzerActions.SyntaxNodeActionsCount;
-                Counts[7] = analyzerActions.SyntaxTreeActionsCount;
+                Counts[0] = analyzerActions.CodeBlockActionsCount;
```

Comment:
Would changing the order of these actions interfere with telemetry that we've already collected? If so, we should probably just put the new actions at the end. @heejaechang would know.